Makefile.smallos

```diff
@@ -0,0 +1,73 @@
+# Makefile magic to determine the Soletta directory based on this very file.
```

Comment:
On Mon, 04 Apr 2016 14:23:30 -0700, Gustavo Sverzut Barbieri wrote:
> > @@ -0,0 +1,73 @@
> > +# Makefile magic to determine the Soletta directory based on this very file.
> 
> add copyright?

Really? We don't have copyright on any Makefile.